README.md

```diff
@@ -32,6 +32,9 @@ Note that Windmill also designates a `u/` directory for storing code per user on
 
 For information on developing scripts, see the [Windmill Scripts quickstart](https://www.windmill.dev/docs/getting_started/scripts_quickstart).
 
+While it is possible in Windmill to [share common logic across scripts with relative imports](https://www.
```

Comment:
avoid line break in the URL.  Markdown won't render it right.

Reply:
Oops, a typo.